### ISSUE TITLE
"osc login" generates bad .config file

### ISSUE THREAD

**Issue by thoraxe** [openshift/origin#1356]
After running:

````
osc login -n demo --certificate-authority=/var/lib/openshift/openshift.local.certificates/ca/cert.crt --server=https://ose3-master.example.com:8443
```

The following is put in `~/.config/openshift/.config`:

```
    apiVersion: v1
    clusters:
    - cluster:
        certificate-authority: /var/lib/openshift/openshift.local.certificates/ca/cert.crt
        certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM1akNDQWRDZ0F3SUJBZ0lCQVRBTEJna3Foa2lHOXcwQkFRc3dKakVrTUNJR0ExVUVBd3diYjNCbGJuTm8KYVdaMExYTnBaMjVsY2tBeE5ESTJOamc1TXpNek1CNFhEVEUxTURNeE9ERTBNelV6TWxvWERURTJNRE14TnpFMApNelV6TTFvd0pqRWtNQ0lHQTFVRUF3d2JiM0JsYm5Ob2FXWjBMWE5wWjI1bGNrQXhOREkyTmpnNU16TXpNSUlCCklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUFyZCtZQnlXSkRRdzN6ZklMOXF2UHg3bzcKd1oxQmhMamJPTWhnQ29WQUhjNEMzUjE0b2dxZ2RjZDFwc1JEamxFa1orYU1TemlyaXg0RVRKNHE3TW9uOHQ2TgpsaElWMlJmaXo0SEQ1Mm5wOThhaU5qeDY1ZnpPNGE4aHJTMDZOQmg1dWREdTJnalp3d0dyeFVNdW5jbzExRS9VCmFGNitheVhvaitOS01kak5leHpmM0hNR2Vld2tzd3o0Z3kxd29RdDFKMWplS21JVHVXaHA2a1BRUitqTy9CbEIKN2duRG1HL0pVWi8zUkJlNmxqRHRBblBXSGROMlYxcUVHbENEdkF6dVRYVVJTU1VjMS9udEdUbyt1TzhFQ1pURgp5N3FHbVU4QkVqYTM4VklVYTltLzlNcHVQdlptV05Ed1N1YWtKOGlsRHQ4Y0NHczcrMUU0MDR3anhZS1pVUUlECkFRQUJveU13SVRBT0JnTlZIUThCQWY4RUJBTUNBS1F3RHdZRFZSMFRBUUgvQkFVd0F3RUIvekFMQmdrcWhraUcKOXcwQkFRc0RnZ0VCQUFIOGhNNUMzVjkwUVNyK1A4a3lpTDl0UHNJNUVsZlQxV1BsbTVVRHVnKzVaVkpSVk95WgpvektaSDBSWmR3TVdtSkhtVEJuRXd3V3pGVzVkc0NUQWJyZzBIWEVLcUxyNkxZZFhtMER1bVp1QkxxWGZQSHlBClBJcENleERvQW1Ub0ZDdG0ySkwxcU84YVI2TXBBdkQ3d0FsdTZDdlRIMWtYSGhUQ1hPQVMyVUFQY0drNUhzejMKOFpFTTUwSktpMForcnJTcy9GK2prU1BEU083K2JFZ3lnR3BUckVTbmdmZVBVS3E3K3JGS2VIRHRmVXUrKzNsOQpQeFhvUlRIanNSSDV2NkRIWTNNUFVjYndSQWJRNWEyWkYyRkZ1QTBKdC9DWW1Yc0NCejFCOGh6RnFvQkxZWkNrCmRQREtGVC9IVmRyTmlxNExLL0ltbmRJVXVUVkoraU5nZitJPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
        server: https://ose3-master.example.com:8443
      name: ose3-master.example.com:8443
    contexts:
    - context:
        cluster: ose3-master.example.com:8443
        namespace: demo
        user: joe
      name: ose3-master.example.com:8443-joe
    current-context: ose3-master.example.com:8443-joe
    kind: Config
    preferences: {}
    users:
    - name: joe
      user:
        token: ZDE3ODY2MWQtYThlYi00NjlhLThiNDEtYmE3OGIzY2Y4YmNk
```

Any further osc commands then give the following error:

```
F0318 12:24:07.917995    6211 get.go:84] certificate-authority-data and certificate-authority are both specified for ose3-master.example.com:8443. certificate-authority-data will override.
```

Removing one or the other of the mentioned items will fix the problem, but it would appear that login is generating a bad config file.

**Comment by fabianofranz** [comment]
Fixed in https://github.com/openshift/origin/pull/1360

**Comment by thoraxe** [comment]
This does not appear to be fixed by #1360
